src/plonk/circuit.rs

```diff
@@ -1187,17 +1193,21 @@ impl<F: Field> ConstraintSystem<F> {
         panic!("get_instance_query_index called for non-existent query");
     }
 
-    pub(crate) fn get_any_query_index(&self, column: Column<Any>, at: Rotation) -> usize {
+    pub(crate) fn get_any_query_index<C: Into<Column<Any>> + Copy>(
+        &self,
+        column: C,
+        at: Rotation,
+    ) -> usize {
+        let column = column.into();
```

Comment:
Likewise, this is a crate-private method, so doesn't need to be altered in this PR (since we're only making changes to improve the public API):
```suggestion
    pub(crate) fn get_any_query_index(&self, column: Column<Any>, at: Rotation) -> usize {
```